testing/frr/APKBUILD

```diff
@@ -0,0 +1,73 @@
+# Contributor: Arthur Jones <arthur.jones@riverbed.com>
+pkgname=frr
+pkgver=5.0
+pkgrel=0
+pkgdesc="Free Range Routing is a fork of quagga"
+url="https://frrouting.org/"
+arch="x86_64"
```

Comment:
Is it really limited to x86_64? I am asking because we are currently building quagga for a lot more archs - https://pkgs.alpinelinux.org/packages?name=quagga&branch=edge

Reply:
I don't think it's limited to x86_64, it should support all architectures, but I only have x86_64 setup for testing.